src/sql/workbench/contrib/dashboard/browser/contents/widgetContent.component.ts

```diff
@@ -104,7 +104,7 @@ export class WidgetContent extends AngularDisposable implements AfterViewInit {
 		'auto_resize': false,       //  Automatically set col_width/row_height so that max_cols/max_rows fills the screen. Only has effect is max_cols or max_rows is set
 		'maintain_ratio': false,    //  Attempts to maintain aspect ratio based on the colWidth/rowHeight values set in the config
 		'prefer_new': false,        //  When adding new items, will use that items position ahead of existing items
-		'limit_to_screen': true,   //  When resizing the screen, with this true and auto_resize false, the grid will re-arrange to fit the screen size. Please note, at present this only works with cascade direction up.
+		'limit_to_screen': false,   //  When resizing the screen, with this true and auto_resize false, the grid will re-arrange to fit the screen size. Please note, at present this only works with cascade direction up.
```

Comment:
if you could plz update the comment as per the new change - will be great

Reply:
this will cause widgets to not go smaller - right?

Reply:
> > @udeeshagautam does this look good to you?
> 
> Looks good! thanks!
> To understand better on the description - the widgets will NOT go smaller when page size is reduced, but will be available to view through scroll - right?
Yes, you will be able to scroll vertically and horizontally.

Reply:
yes, this will make the widgets to maintain their original size. and the comments is still correct.